### PR TITLE
Mobile nav V1

### PR DESCRIPTION
**Pull request by Ruben-Brits**

### CODE REVIEW

File: libs/explore/feature/src/lib/explore.component.html [hunk 1]
```diff
@@ -45,41 +63,48 @@
...
+                            <ion-label class="slider-icons" slot="start">Max:</ion-label>
+                        </ion-range>
+                        <input type="submit" class="filter-submit-button" id="change" value="APPLY" />
+                    </form>
+                </div>
+            </div>
         </ion-content>
     </ion-menu>
-</ion-content>
+</ion-content>
```
Comment (PxttytheThxtty) [inline libs/explore/feature/src/lib/explore.component.html:110]: Newline

File: libs/shared/components/navbar/src/lib/navbar.component.html [hunk 2]
```diff
@@ -16,4 +16,94 @@
...
+        <ion-col  class="menu-item-text" size="10">
+          <ion-text color="light">
+            Logout
+          </ion-text>
+        </ion-col>
+      </ion-row>
+    </ion-card>
+
+  </ion-content>
+</ion-menu>
```
Comment (PxttytheThxtty) [inline libs/shared/components/navbar/src/lib/navbar.component.html:109]: Newline

File: libs/shared/components/navbar/src/lib/navbar.component.scss [hunk 3]
```diff
@@ -66,3 +66,61 @@ h1{
...
+
+ion-menu::part(backdrop) {
+    background-color: transparent;
+    display: none;
+    
+}
+
+.menu-wrapper{
+    margin-bottom: 40px;
+}
```
Comment (PxttytheThxtty) [inline libs/shared/components/navbar/src/lib/navbar.component.scss:126]: Newline

**Review by PxttytheThxtty — DISMISSED** [review]

**Review by PxttytheThxtty — APPROVED** [review]

File: libs/shared/components/navbar/src/lib/navbar.component.html [hunk 4]
```diff
@@ -16,4 +16,94 @@
       </ion-buttons>
       <ion-title><h1 class="text-white ion-text-center">The Au Pair</h1></ion-title>
     </ion-toolbar>
+
+
```
Comment (ricecakesreece) [inline libs/shared/components/navbar/src/lib/navbar.component.html:20]: extra line

File: libs/shared/components/navbar/src/lib/navbar.component.ts [hunk 5]
```diff
@@ -56,20 +61,16 @@ export class NavbarComponent implements OnInit
...
-    }
-    else if(this.type == 1)
-    {
-      this.router.navigate(['/parent-dashboard']);
-    }
-    else if(this.type == 2)
-    {
-      this.router.navigate(['/au-pair-dashboard']);
-    }
+    console.log("menu open");
```
Comment (ricecakesreece) [inline libs/shared/components/navbar/src/lib/navbar.component.ts:66]: remove console.log

File: libs/shared/components/navbar/src/lib/navbar.component.ts [hunk 6]
```diff
@@ -85,4 +86,29 @@ export class NavbarComponent implements OnInit
       this.router.navigate(['/admin-reports']);
     }
   }
+
+  explore(){
+    if (this.kids < 1){
+      this.openToast('You need to have children added to your profile in order to hire an Au Pair');
```
Comment (ricecakesreece) [inline libs/shared/components/navbar/src/lib/navbar.component.ts:92]: Does the length of the sentence work okay with our toast?

Comment (Ruben-Brits) [reply libs/shared/components/navbar/src/lib/navbar.component.ts:92]: This is copied and pasted from our other toast from dashboard

Comment (ricecakesreece) [reply libs/shared/components/navbar/src/lib/navbar.component.ts:92]: Okay cool should be good then

**Review by priyo5 — APPROVED** [review]

**Pull request merged**